httpclient5/src/main/java/org/apache/hc/client5/http/ssl/DefaultHostnameVerifier.java

```diff
@@ -228,36 +229,44 @@ private static boolean matchIdentity(final String host, final String identity,
                                          final PublicSuffixMatcher publicSuffixMatcher,
                                          final DomainType domainType,
                                          final boolean strict) {
+
+        final String unicodeIdentity;
```

Comment:
`unicodeIdentity` -> `normalizedIdentity` would be a better name?

Reply:
Done